feature/experimental/p4rt/ate_tests/traceroute_packetin_test/traceroute_packetin_test.go

```diff
@@ -0,0 +1,377 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package traceroute_packetin_test
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/wbb"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+var (
+	p4InfoFile            = flag.String("p4info_file_location", "../../wbb.p4info.pb.txt", "Path to the p4info file.")
+	p4rtNodeName          = flag.String("p4rt_node_name", "FPC0:NPU0", "component name for P4RT Node")
+	streamName            = "p4rt"
+	deviceId              = *ygot.Uint64(1)
+	portId                = *ygot.Uint32(10)
+	electionId            = *ygot.Uint64(100)
+	METADATA_INGRESS_PORT = uint32(10)
+	METADATA_EGRESS_PORT  = uint32(0)
+	TTL1                  = uint8(1)
+	HopLimit1	      = uint8(1)
+	TTL0                  = uint8(0)
+	HopLimit0	      = uint8(0)
+	ipv4PrefixLen	  = uint8(30)
+	ipv6PrefixLen	  = uint8(12)
```

Comment:
ipv6PrefixLen needs to be changed

Reply:
Thanks @mingyangcisco for all your input. I'm working on next commit with all changes you mentioned.

Reply:
change suggested are incorporated in 10/06 commit.